index.html

```diff
@@ -27,13 +33,17 @@ <h1>I'm Mustafa <br> Glad to see you!</h1>
                 <i class='fa fa-twitter'></i>
                 <i class='fa fa-linkedin'></i>
                 <i class='fa fa-medium'></i>
-                <i class='fa fa-angellist'></i>
                 <i class='fa fa-github'></i>
+                <i class='fa fa-angellist'></i>
             </ul>
         </section>
         <section class="works">
             <div class="card1 card">
-                <img src="/imgs/Snapshoot Portfolio(2).png" alt="">
+                <div class="card-img">
+                    <img src="/imgs/Snapshoot Portfolio(2).png" alt="">
+                </div>
+                <!-- card one -->
+               <div class="card-info">
```

Comment:
Great job on your project cards! I have a few suggestions to help make them even better:

- [ ] Would you mind checking the `font-size` of your project card titles? They appear **slightly smaller than the [given template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122).** Adjusting the `font-size` **will ensure better consistency with the desired design.** 🔍

- [ ] Could you **kindly review the size of your image?** It seems a bit smaller compared to the template. Perhaps reducing the `height` of the cards would help achieve the desired look. 🤓

- [ ] Please check the font weight and colors of your **"CANOPY," "Back End Dev," and "2015"** elements to align them with the [given template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122). Additionally, please note that **there should be a circular icon between these elements.** 🙌

- [ ] Would you mind checking the horizontal padding of your buttons? Also, please review the `font-weight` of the text within the buttons to align them with the given template. 👍

   **_These suggestions aim to enhance the overall appearance of your project cards, ensuring they closely resemble the desired template. Thank you for considering these improvements! 😉_**


   ### Your design:
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/8ea4d07d-bc50-44c3-9a09-4c82edf4ae20)

   ### Figma design:
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/6ff7f41f-070a-4d76-9a5d-9b406b8e2541)
   
   _Just a friendly reminder that following the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122) is a key requirement for this project. You're doing a great job so far, and it's important to keep the template in mind as you continue with your design to ensure that your project meets all of the project requirements._
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/9650a956-df52-488a-b9c1-dd120581b502)

Reply:
resolved